src/ngraph/runtime/backend.cpp

```diff
@@ -29,9 +29,9 @@ runtime::Backend::~Backend()
 {
 }
 
-std::shared_ptr<ngraph::Node> runtime::Backend::get_backend_node(const std::string& node_name, ...)
+std::shared_ptr<ngraph::op::Op> runtime::Backend::get_backend_op(const std::string& op_name, ...)
```

Comment:
Safer to return `std::shared_ptr<Node>` or you're likely to encounter a C++ types issue. Just call the function `get_backend_op` but leave it returning a Node.

Reply:
@diyessi since ngraph::Op is a derived class of ngraph::Node - what `you're likely to encounter a C++ types issue` are you referring to?